db-models/listings.sql

```diff
@@ -0,0 +1,14 @@
+-- db-models/listings.sql
+CREATE TABLE listings (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,       -- Auto-incrementing ID
+    author TEXT NOT NULL,                       
+    title TEXT NOT NULL,                     
+    price  NOT NULL,                            -- Floating-point number
+    item_description TEXT NOT NULL,              
+    seller TEXT NOT NULL,                       
+    seller_id INTEGER NOT NULL,                 -- Foreign key to link to the sellers table
+    class TEXT NOT NULL,                        
```

Comment:
Let's make this `class_subject` or something similar, just to make it more obvious this attribute is like `HC`, `MTH`, `CSI`, etc. and not strictly `APM 2663`